_data/risc_outgoing.yml

```
@@ -112,3 +129,20 @@
       }
     }
   }
+
+- friendly_name: Reproofing Completed
+  event_type: https://schemas.login.gov/secevent/risc/event-type/reproof-completed
+  spec_url:
+  description: >
+    Login.gov pushes this event when a user reproofs after having an active verified profile.
```

I feel like "reproof" is a word we invented, and "proofing" is kind of jargony, what about something like this?

```suggestion
    Login.gov pushes this event when a user re-verifies their identity (goes through the identity proofing process). This event is only sent if they previously had an active verified profile, not for the first-time identity verification.
```